### PR TITLE
Initial setup of jest and enzyme for testing

### PR DESCRIPTION
**Pull request by eLeiLei**
- Install dependencies
- Update configurations

### CODE REVIEW

**Comment by eLeiLei** [comment]
Cleaning up the PR.

**Comment by eLeiLei** [comment]
Added files .gitignore

File: client/index.jsx [hunk 1]
```diff
@@ -1,40 +1,5 @@
...
-    return (
-      <div>
-        index.jsx: I will eventually evolve into a photo..
-        <p />
-        <a href={link} onClick={this.getPhotosById}>Record 10</a>
-      </div>
-    );
-  }
-}
+import PhotoBanner from './components/PhotoBanner.jsx';
```
Comment (jessebeard) [inline client/index.jsx:3]: you didn't add this file to your commit!

File: package.json [hunk 2]
```diff
@@ -22,17 +25,24 @@
   },
   "devDependencies": {
     "@babel/core": "^7.9.6",
+    "@babel/polyfill": "^7.8.7",
```
[the remark this answers is not in the document]
Comment (eLeiLei) [reply package.json:28]: Thanks. Added a ticket here [https://trello.com/c/qcIyMYkJ](https://trello.com/c/qcIyMYkJ). I plan to revisit that when writing the first unit test for the front-end code.

File: client/index.jsx (hunk 1 above)
Comment (eLeiLei) [reply client/index.jsx:3]: Will add!

**Pull request merged**